docs/source/reference_api/index.rst

```diff
@@ -0,0 +1,37 @@
+Reference APIs
```

Comment:
Shall this change be reflected in the commit message as this is not specific to ad405x doco?